index.js

```diff
@@ -24,12 +24,12 @@ module.exports = {
 		'/**/resources/',
 		'/**/resources/*'
 	],
-	parser: 'babel-eslint',
+	parser: '@babel/eslint-parser',
 	parserOptions: {
-		ecmaVersion: 2018,
 		sourceType: 'module',
-		ecmaFeatures: {
-			jsx: true
+		requireConfigFile: false,
+		babelOptions: {
+			presets: [require.resolve('babel-preset-react-app/prod')]
```

Comment:
As I told, let's replace this to our babel rc file.

Reply:
For resolving this problem, I've added 3 plugins that enact babel config uses.
				require('@babel/plugin-proposal-export-default-from').default,
				require('@babel/plugin-proposal-export-namespace-from').default,
				require('@babel/plugin-syntax-dynamic-import').default
New `@babel/plugin-proposal-export-default-from` added in devDependencies
(others are already included in babel-preset-react-app/node_modules/@babel/preset-env)